Dockerfile.ci-portal

```diff
@@ -0,0 +1,13 @@
+ARG REGISTRY
+
+FROM ${REGISTRY}/ubi8/nodejs-16 AS portal-base
+WORKDIR /build/portal/v2
+USER root
+
+FROM portal-base AS portal-build
+COPY /portal/v2/package*.json ./
+RUN npm ci
+RUN npm audit --omit=dev
+COPY /portal/v2/ ./
+RUN npm run lint
+RUN npm run build
```

Comment:
I'm not too familiar with JS and npm, but I was tracing through the build process using the links you provided in the PR description, and I noticed that the ["old" build command](https://github.com/Azure/ARO-RP/blob/f20df8471736ba501a9bf86a660451e8dd26a5ac/Makefile#L162) includes an `npm install` that's not in this Containerfile. Why'd you remove that?

Reply:
`npm install` is a near-duplicate (dev version) of `npm ci`, which performs all the actions of `npm install` but without any caching. Honestly, considering how the containerfile is written (only copy `package.json` files, then install deps) there wouldn't be a cache to pull from anyways... but `npm ci` is a more robust version of `npm install` in any case, so I chose it instead.